### ISSUE TITLE
Failed to start session message

### ISSUE THREAD

**Issue by fossfreedom** [BuddiesOfBudgie/budgie-desktop#173]
## Description
On a fresh install with git master as per 28th Jul I get the above message displayed by slick-greeter

## Budgie version
10.6.2 - git master 28th July

## Operating system
Ubuntu Budgie 22.10

## Steps to reproduce the issue
Try to login

## Actual result
Above message

## Expected result
Desktop should be shown

## Additional information
Renaming /usr/share/xsessions/org.buddiesofbudgie/budgie-desktop.desktop back to "budgie-desktop.desktop" allows the desktop to load.

So not sure why (at least on UB) the xsession needs to be "budgie-desktop.desktop" as opposed to the new 'reverse DNS' notation.
Will dig around. Worse case either revert that part from git master - or I will just distro patch if its a UB specific thing.

**Comment by serebit** [comment]
I think I got that too after the upgrade, but it may have been because GNOME is autoselected after a reboot even though the required components aren't installed? Not sure.

**Comment by fossfreedom** [comment]
Yeah - not sure either.  This is a single session installation - i.e. just budgie installed.

I've checked that my /etc/lightdm/lightdm.conf.d/50_budgie.conf file  has the correct new session name (old or new) doesnt make a difference.

The other issue I found digging around is that the environment variable $DESKTOP_SESSION has now changed.  So that will obviously break anything such as scripts dependent on that var.
